.github/workflows/ci.yaml

```diff
@@ -136,11 +111,12 @@ jobs:
             echo Skipping unstable tests
             echo "mark_expression=not unstable" >> $GITHUB_OUTPUT
           fi
+
       - name: Run integration tests
         run: |
-          # set sysctl values in case the cloudinit-userdata not applied
-          sudo sysctl -w vm.max_map_count=262144 vm.swappiness=0 net.ipv4.tcp_retries2=5
-
+          # set up environment - get juju version and set kernel params
+          echo ${{matrix.runner}}
+          source ./setup.sh
```

Comment:
Do we need to hide two bash commands into the script instead of running them explicitly here?